pkg/filesystem/subvolume.go

```diff
@@ -178,12 +183,28 @@ func Delete(ctx context.Context, clientsets *k8sutil.Clientsets, OperatorNamespa
 				continue
 			}
 			logging.Info("subvolume %q deleted", subvolume)
+			DeleteOmap(ctx, clientsets, OperatorNamespace, CephClusterNamespace, subvolume, fs)
 		} else {
 			logging.Info("subvolume %q is not stale", subvolume)
 		}
 	}
 }
 
+// DeleteOmap deletes omap object and key for the given subvolume.
+func DeleteOmap(ctx context.Context, clientsets *k8sutil.Clientsets, OperatorNamespace, CephClusterNamespace, subVol, fs string) {
+	omapdetails, err := getOmapDetails(ctx, clientsets, OperatorNamespace, CephClusterNamespace, subVol, fs)
+	if err != nil {
+		logging.Fatal(err)
```

Comment:
```suggestion
		logging.Fatal(err)
```

it's better to good error message along with error